src/pages/Schedule/index.jsx

```diff
@@ -0,0 +1,167 @@
+import React, { useState } from "react";
+import Typography from "@mui/material/Typography";
+import { Helmet } from "react-helmet";
+import Button from "@mui/material/Button";
+import DeleteIcon from "@mui/icons-material/Delete";
+import { useLocalStorage } from 'usehooks-ts';
+
+export default function Schedule() {
+  const [inputValue, setInputValue] = useLocalStorage("inputValue", "");
+  const [checkboxData, setCheckboxData] = useLocalStorage("checkboxData",[
+    { name: "ceph-repo", value: "https://github.com/ceph/ceph-ci.git", checked: false },
+    { name: "suite-repo", value: "https://github.com/ceph/ceph-ci.git", checked: false },
+    { name: "ceph-branch", value: "main", checked: false },
+    { name: "subset", value: "1/4", checked: false },
+  ]);
+  const [hoveredRowIndex, setHoveredRowIndex] = useState(null);
+  const [checkedRows, setCheckedRows] = useLocalStorage("checkedRows", []);
+
+  const handleInputChange = (event) => {
+    setInputValue(event.target.value);
+  };
+
+  const handleRun = () => {
+    updateCommand(checkedRows);
+  };
+
+  const handleDryRun = () => {
+    // setDryRunOutput(inputValue + " (Dry Run)");
+  };
```

Comment:
is the implementation missing, or is the function unnecessary?

Reply:
Yes, need to implement dryrun logic here